azure/packages/test/end-to-end-tests/.npmignore

```diff
@@ -2,3 +2,5 @@ nyc
 *.log
 **/*.tsbuildinfo
 **/_api-extractor-temp/**
+src/test
+dist/test
```

Comment:
Was this part of a warning? I'm not the most versed in this part of the codebase, but I think this might break running the tests in this package. Basically all of the test files are in src/test, so if they're not published with the npm package, I can imagine this having weird consequences. I looked at `packages/test/test-end-to-end-tests/.npmignore` and we don't ignore these folders there either. For a "regular" library package it makes sense, tests shouldn't be in the published output, but for this one I think it's fine (maybe even necessary?).

Reply:
Unfortunately it is part of a warning because it's a test package with a prefix that isn't part of the allow-list in fluid-build. I'm removing this from the .npmignore at the cost of adding some warnings back in. Good catch!